java/code/src/com/suse/manager/virtualization/VirtManagerSalt.java

```diff
@@ -38,6 +44,8 @@
 public class VirtManagerSalt implements VirtManager {
```

Comment:
Shouldn't this be called "SaltVirtManager" ?

Reply:
Well, that class has been named like this by @lucidd during the refactoring.

Reply:
ok, maybe for a next PR then.

Reply:
I think we don't really have a written rule or a scheme for the naming of interfaces and implementations yet? For instance I recently created `MonitoringManager` (interface) and `FormulaMonitoringManager` (implementation) and did the naming the other way around. I would have been fine though to align with another scheme, so I guess we could maybe agree on a scheme and write it down?